### ISSUE TITLE
Unable to resolve modules after upgrading from 0.47

### ISSUE THREAD

**Issue by rajivshah3** [facebook/react-native#17557]
### Is this a bug report?

Yes

### Have you read the [Contributing Guidelines](https://facebook.github.io/react-native/docs/contributing.html)?

Yes

### Environment

Environment:
  OS: macOS High Sierra 10.13.2
  Node: 8.5.0
  Yarn: 1.3.2
  npm: 5.6.0
  Watchman: 4.9.0
  Xcode: Xcode 9.2 Build version 9C34b
  Android Studio: 3.0 AI-171.4408382

Packages: (wanted => installed)
  react: 16.0.0 => 16.0.0
  react-native: ^0.52.0 => 0.52.0

### Steps to Reproduce

1. Upgrade from react-native 0.47
2. Attempt to run the project on iOS

### Expected Behavior

Build successfuly

### Actual Behavior

![image](https://user-images.githubusercontent.com/19519564/34860053-cd0bacb2-f728-11e7-90ff-1cbd8571f2e8.png)

### Reproducible Demo

The following modules seem to have given me some issues:

- XMLHttpRequest (specifically url, http, and child_process)
- redux
- crypto (we use rn-nodeify to fix this one)

I've tried removing the node_modules and installing again, clearing the watchman cache, removing the temp directories, and pretty much anything else I've found online

**Comment by jinxac** [comment]
@rajivshah3 is any other process running on port 8081? Can you verify and kill that process and restart your server?

**Comment by rajivshah3** [comment]
@jinxac I see this: 
```
Rajivs-MBP:~ rajiv$ sudo lsof -i tcp:8081
COMMAND   PID  USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
node    86711 rajiv   20u  IPv6 0x4f5d6b36a32cc6cd      0t0  TCP *:sunproxyadmin (LISTEN)
```
But I always assumed it had something to do with the packager

**Comment by tracinah** [comment]
Why not?
Get organized with Yahoo Mail 
 
  On Tue, Jan 23, 2018 at 11:53 PM, Rajiv Shah<notifications@github.com> wrote:   

Closed #17557.
